presentation/backend/src/test/ElementPropertiesHelper.test.ts

```diff
@@ -63,6 +64,58 @@ describe("getElementsCount", () => {
   });
 });
 
+describe("createIdBatches", () => {
+  it("returns empty list when given no ids", async () => {
+    expect(await firstValueFrom(createIdBatches([], 2).pipe(toArray()))).to.be.deep.eq([]);
+  });
+
+  it("creates a batch larger than element ids count, when there's one element id", async () => {
```

Comment:
What does `creates a batch larger than element ids count` means in this context?

Reply:
It means requested batch size is larger than the number of elements. Renamed some tests to clarify that.